### PR TITLE
Feature/create planet divs

### PR DESCRIPTION
**Pull request by hdpuk86**
All the circles are in with their name and info divs hidden but there's not listeners yet and they are not spaced

### CODE REVIEW

**Review by Findsay — CHANGES_REQUESTED** [review]
I think you've accidentally copied in the UI file from films. We could probably refactor so we could  reuse the planetView functions(maybe passing in the id and class names) to keep with DRY but i'm fine putting this in to get something working and then working on refactoring :)

**Comment by Findsay** [comment]
Your one step ahead of me! Happy to proceed with this for now :)

**Review by Findsay — APPROVED** [review]

File: client/src/views/hoverView.js [hunk 1]
```diff
@@ -27,8 +25,8 @@ function createPlanetStats(planet){
   return ul;
 };
 
-var HoverView = function(planet){
+var Hover = function(planet){
   return createHoverDiv(planet);
 };
 
-module.exports = HoverView;
+module.exports = Hover;
```
Comment (JamieSK) [inline client/src/views/hoverView.js]: Why have you done this rather than `module.exports = createHoverDiv`?

**Review by JamieSK — APPROVED** [review]
I've left one comment on the code but it's not a big deal, nice work.

File: client/src/views/hoverView.js (hunk 1 above)
Comment (hdpuk86) [reply client/src/views/hoverView.js]: not sure I was following it all through and must have got confused, I changed it back

**Pull request merged**